packages/angular/build/src/tools/vite/id-prefix-plugin.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.dev/license
+ */
+
+import type { Plugin } from 'vite';
+
+// NOTE: the implementation for this Vite plugin is roughly based on:
+// https://github.com/MilanKovacic/vite-plugin-externalize-dependencies
+
+const VITE_ID_PREFIX = '/@id/';
+const vitePrefixRegex = new RegExp(VITE_ID_PREFIX, 'g');
```

Comment:
The `g` flag isn't used, I believe

Reply:
updated: this regex is removed now, based on the earlier comment by Charles.